### PR TITLE
Better UI for Signup and Login page

### PR DESCRIPTION
**Pull request by Ankita297**
## Description
- styling of signup and login page updated
- #50 

## Affected Dependencies
NA

## How has this been tested?
I have tested it in my local  system .

## GIF Before

https://user-images.githubusercontent.com/67833077/111673259-ccc9e500-8840-11eb-81ef-6745e821293e.mp4

## GIF After

https://user-images.githubusercontent.com/67833077/112419823-17bc8e80-8d52-11eb-907e-e5e7676c53b7.mp4

https://user-images.githubusercontent.com/67833077/112419681-d330f300-8d51-11eb-8616-6c05dc55905e.mp4

## Checklist
- [ x] I have followed the [Contribution Guidelines](https://github.com/Praful932/Kitabe/blob/master/CONTRIBUTING.md).
- [ x] My changes do not edit/add any unrequired files.
- [x ] My changes are covered by tests.

### CODE REVIEW

**Review by maneprajakta — CHANGES_REQUESTED** [review]
The hovering effect added doesn't seem good to me. Can you improve it?

**Comment by Ankita297** [comment]
@maneprajakta  should i remove it ?

**Comment by Ankita297** [comment]
or any other idea ?

**Comment by maneprajakta** [comment]
> @maneprajakta should i remove it ?

Yep remove it.

**Comment by Ankita297** [comment]
Is it okay now @maneprajakta ?

**Comment by maneprajakta** [comment]
> Is it okay now @maneprajakta ?

@Ankita297 the color doesn't go with our website theme.Try to match with it.

**Comment by Ankita297** [comment]
@maneprajakta  are you talking about form color . should i change it color to blue like in read genre wise section or any other idea from your side .

**Comment by maneprajakta** [comment]
@Ankita297 
Keep it white, remove the grey color also decrease the space between sign in  and sign up button 
You can improve the button style their color any color which goes with the background.

**Comment by Ankita297** [comment]
Sign Up - Google Chrome 18-03-2021 21_37_59
Sign Up - Google Chrome 18-03-2021 21_38_06

**Comment by Ankita297** [comment]
is it okay now @maneprajakta

**Comment by maneprajakta** [comment]
> is it okay now @maneprajakta

Yep :100:

**Review by Praful932 — CHANGES_REQUESTED** [review]
@Ankita297 Add GIFS of UI in responsive mode in all sizes, check earlier page to see how it behaves with change in screen size

**Comment by Ankita297** [comment]
@Praful932 @maneprajakta  is it okay now ?

**Comment by maneprajakta** [comment]
@Ankita297 you need to wait for a while we have PR #117 we need to merge that. The PR will add a button to a sign-up page.
After that, you need to update your fork do changes for that button only. We will let you know.

**Comment by Praful932** [comment]
@Ankita297 You can proceed to update your branch with the new changes, the required PR has been pushed

**Comment by Ankita297** [comment]
@Praful932 Done Sir .

**Review by maneprajakta — CHANGES_REQUESTED** [review]
@Ankita297 Could you place `Sign Up With Google` and `Sign In With Google`  both buttons inside the form border below sign-in button and keep its styling same as sign-in button .
For both pages do same.

**Comment by Ankita297** [comment]
Sign Up - Google Chrome 20-03-2021 13_06_25

**Comment by Ankita297** [comment]
@maneprajakta  is it okay ?

**Comment by Praful932** [comment]
> @Ankita297 Could you place `Sign Up With Google` and `Sign In With Google` both buttons inside the form border below sign-in button and keep its styling same as sign-in button .
> For both pages do same.

We merged that PR, @Ankita297 needs to update her branch,
@Ankita297 check the current latest [page](https://kitabe-app.herokuapp.com/accounts/login/), Also no need to call sir :)

**Comment by maneprajakta** [comment]
> @maneprajakta is it okay ?

Yep :rocket:

**Comment by Ankita297** [comment]
@maneprajakta  Done.

File: templates/account/login.html [hunk 1]
```diff
@@ -41,9 +41,11 @@ <h3 class="welcome">Welcome Back!</h3>
                     {% endif %}
                     <button class="loginbutton primaryAction" type="submit">{% trans "Sign In" %}</button>
                     <br>
+
+                    <a href="{% provider_login_url 'google' %}"><button class="googlebutton">Sign In with Google</button></a>
```
Comment (maneprajakta) [inline templates/account/login.html:45]: the button doesn't work now

File: templates/account/signup.html [hunk 2]
```diff
@@ -43,10 +43,10 @@ <h3 class="welcome">Welcome!</h3>
                     {% endif %}
                     <button class="signupbutton" type="submit">{% trans "Sign Up" %}</button>
                     <br>
-    
+                    <a href="{% provider_login_url 'google' %}"><button class="googlebutton">Sign Up with Google</button></a>
```
Comment (maneprajakta) [inline templates/account/signup.html:46]: The button doesn't work

**Review by maneprajakta — CHANGES_REQUESTED** [review]
- For small screen, it's messed up make it responsive
Screenshot from 2021-03-20 22-09-09
Screenshot from 2021-03-20 22-08-48

- For big screen bottom border of form is not there in Signup.html
Screenshot from 2021-03-20 22-09-22

File: templates/account/signup.html (hunk 2 above)
Comment (Ankita297) [reply templates/account/signup.html:46]: I will resolve it @maneprajakta

**Comment by Ankita297** [comment]
@maneprajakta  check it now .

**Comment by maneprajakta** [comment]
@Ankita297 for small screen it's still messed up.

**Comment by maneprajakta** [comment]
@Ankita297 for small screen it's messed up

**Comment by Ankita297** [comment]
Login - Google Chrome 22-03-2021 08_54_36

**Comment by maneprajakta** [comment]
follow this 360X640 px

**Comment by Ankita297** [comment]
okay

**Comment by Ankita297** [comment]
Login - Google Chrome 22-03-2021 09_20_03

**Comment by Ankita297** [comment]
like his @maneprajakta

**Comment by maneprajakta** [comment]
@Ankita297 yep :+1:

**Review by Praful932 — CHANGES_REQUESTED** [review]
@Ankita297 Please see the deployment

**Review by Praful932 — CHANGES_REQUESTED** [review]
@Ankita297 Please see the deployment, make the `Welcome Back` smaller as in original and move the buttons down
image

**Comment by Ankita297** [comment]
Sir which button should i move down ? Are you asking for sign up with google out of box

**Comment by Ankita297** [comment]
@Praful932

**Comment by Ankita297** [comment]
@Praful932  I have to make the font  of welcome smaller and then which button on which page ?

**Comment by Ankita297** [comment]
@maneprajakta  Please clear it out ?

**Comment by Praful932** [comment]
@maneprajakta Could you have a look at this, the deployement doesn't seem to match the provided ss

**Comment by Ankita297** [comment]
actually @Praful932 I have not udated the  previous gif Do i need to update it

**Comment by Praful932** [comment]
> actually @Praful932 I have not udated the previous gif Do i need to update it

yes

**Comment by Ankita297** [comment]
Okay @Praful932

**Comment by Ankita297** [comment]
@Praful932  is it okay now ?

**Comment by maneprajakta** [comment]
@Ankita297 commit the changes done till now. We will directly check the deployed PR and let you know the necessary chnages.

**Comment by Ankita297** [comment]
@maneprajakta  I have commited all the changes till now

**Review by maneprajakta — CHANGES_REQUESTED** [review]
- Reduce the gap between buttons for both pages in both small & large screens. Marked below by blue. 
Screenshot from 2021-03-23 16-36-42
- Also the border doesn't seem to be centrally aligned it's rightward shifted.

**Comment by Ankita297** [comment]
@maneprajakta   form-group is aligned properly . Please see .

**Comment by maneprajakta** [comment]
Yep :+1:

**Review by Praful932 — CHANGES_REQUESTED** [review]
@Ankita297 It kinda is going out of the screen and also make the whole welcome header a bit smaller see the original page
image

**Comment by Ankita297** [comment]
@maneprajakta  please see

**Review by Praful932 — CHANGES_REQUESTED** [review]
Login seems different now, we need it a bit down, see the original page
image

**Comment by Ankita297** [comment]
Uploading final_605bfc5a40ffe00048a61f18_975245.mp4…

**Comment by Ankita297** [comment]
@maneprajakta  @Praful932  If it is okay from Your side then  i will commit the changes .

**Comment by Ankita297** [comment]
https://user-images.githubusercontent.com/67833077/112412465-8abf0880-8d44-11eb-96fc-185d75ea7bbd.mp4

**Comment by Praful932** [comment]
@Ankita297 Looks better now, you can push the new changes

**Comment by Ankita297** [comment]
@Praful932   Please review .

**Review by Praful932 — APPROVED** [review]

**Comment by Ankita297** [comment]
@maneprajakta Please review the changes .

**Review by maneprajakta — APPROVED** [review]
Great Work :rocket:

**Comment by Ankita297** [comment]
Thankyou @maneprajakta  @Praful932 .

**Pull request merged**